pkg/build/helm_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"helm.sh/helm/v3/pkg/chart"
+	"sigs.k8s.io/yaml"
+
+	"istio.io/release-builder/pkg/model"
+)
+
+func TestHelmUpdate(t *testing.T) {
+	cases := []struct {
+		name            string
+		inputChartfile  string
+		inputValuesfile string
+		inputManifest   model.Manifest
+	}{
+		{
+			"test-version",
+			filepath.Join("testdata", "chart-deps-in.yaml"),
+			filepath.Join("testdata", "chart-values-in.yaml"),
+			model.Manifest{
+				Version: "1.19.13-eks-8df270", // sufficiently oddball version string
+			},
+		},
+	}
+
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			dir, _ := os.MkdirTemp("", tc.name)
+			defer os.RemoveAll(dir)
```

Comment:
nit: t.TempDir does this for you

Reply:
I couldn't remember if the base testing lib did or our wrapper did, yeah ok.